src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Execute.cs

```diff
@@ -20,6 +20,7 @@ internal abstract partial class AbstractOleCommandTarget
     {
         public virtual int Exec(ref Guid pguidCmdGroup, uint commandId, uint executeInformation, IntPtr pvaIn, IntPtr pvaOut)
         {
+            this.CommandID = commandId;
```

Comment:
I see - I'd still rather do something like:
``` C#
try
{
    if (command == GotoBraceExt)
    {
        CurrentlyInGoToBraceExt = true;
    }
   // call the editor
}
finally
{
    CurrentlyInGoToBraceExt = false;
}
```

Or maybe `CurrentlyExecutingCommand` instead of the `bool`.  At any rate, a comment on the property about why it's necessary!